testing/runfossa/runfossa.go

```diff
@@ -3,18 +3,42 @@ package runfossa
 import "github.com/fossas/fossa-cli/exec"
 
 // Init executes fossa init the provided directory
-func Init(projectDir string) error {
+func Init(projectDir string) (string, error) {
+	output, err := runfossa(projectDir, []string{"init"})
+	if err != nil {
+		println(output)
+		return output, err
+	}
+
+	return output, nil
+}
+
+var licenseReportPrefix = []string{"report", "licenses"}
+
+func LicenseReport(projectDir string, args []string) (string, error) {
+	args = append(licenseReportPrefix, args...)
+	return runfossa(projectDir, args)
+}
+
+var dependencyReportPrefix = []string{"report", "dependencies"}
+
+func DependencyReport(projectDir string, args []string) (string, error) {
+	args = append(dependencyReportPrefix, args...)
+	return runfossa(projectDir, args)
+}
+
+func runfossa(projectDir string, argv []string) (string, error) {
 	cmd := exec.Cmd{
-		Argv:    []string{"init"},
+		Argv:    argv,
 		Name:    "fossa",
 		Dir:     projectDir,
 		Command: "fossa",
 	}
 
-	_, errMsg, err := exec.Run(cmd)
+	stdout, stderr, err := exec.Run(cmd)
 	if err != nil {
-		println(errMsg)
+		return stderr, err
```

Comment:
That's a bit alarming. What about the case where `err != nil` and `stdout` is not empty?

That's exactly why `exec.Run` returns 3 arguments. You should probably return all 3 of them.